package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "lwc",
+  "author": "LWC Team",
```

Comment:
should this be a proper github team?

Reply:
not sure if there is any magical auto linking... Dont think is important